src/components/Accordion/Accordion.mdx

```diff
@@ -111,6 +112,19 @@ You may pass any Chakra `Box` props, as well as the props below.
 
 ## Accessibility
 
+<Banner
+  content={
+    <>
+      <strong>NOTE:</strong> It's recommended to pass an `id` value for
+      accessibility requirements. If no `id` is passed, a random `id` will be
+      generated.
```

Comment:
This was a pattern that we used when the DS was very young and we ran into problems with randomly generated values being out of sync in certain testing situations. Will that not be a problem here because of `React.useId()`?